### PR TITLE
Support/add app json upload for mobile

### PR DESCRIPTION
**Pull request by ggilchrist-ledger**
### 📝 Description

Add ability to upload app.json to set account/settings. Steps available in the `start-server.ts` file

### ❓ Context

- **Impacted projects**: LLM, Detox
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/B2CQA-1791

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

https://github.com/LedgerHQ/ledger-live/assets/87659842/a5027f6a-9327-4bf8-8218-58d08a1ef3cb

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/e2e/bridge/start-server.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+3. Run the mobile app in mock mode with: ENVFILE=.env.mock pnpm mobile ios
+4. Every time you reload the app or make a code change in LLM and save it, the app.json data will refresh
+
+*/
+
+import * as serverBridge from "./server";
+
+const filePath = process.argv[2];
+
+console.log("JSON file to look for: ", filePath);
```
Comment (Justkant) [inline apps/ledger-live-mobile/e2e/bridge/start-server.ts:18]: You might want to check if the filePath exist (the string and maybe the file itself too)

File: apps/ledger-live-mobile/e2e/bridge/server.ts [hunk 2]
```diff
@@ -5,12 +5,14 @@ import { NavigatorName } from "../../src/const";
...
 
-export function init(port = 8099) {
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+export function init(port = 8099, onConnection = () => {}) {
   wss = new Server({ port });
   log(`Start listening on localhost:${port}`);
 
   wss.on("connection", ws => {
     log(`Connection`);
+    onConnection();
```
[the remark this answers is not in the document]
Comment (ggilchrist-ledger) [reply apps/ledger-live-mobile/e2e/bridge/server.ts:15]: I think we still need it. When the server receives the connection from the client (LLM) - ie. when the app is loaded - then we want to run `loadConfig` via the bridge server. This function in `init` allows us to start the server with the script and then kick off the `loadConfig` function by passing it in via function parameter. Maybe you mean something else though - if so can you explain?

[the remark this answers is not in the document]
Comment (ggilchrist-ledger) [reply apps/ledger-live-mobile/e2e/bridge/server.ts:15]: I tried something like this but it doesn't work. I think it's not properly waiting for the app to be ready
image

[the remark this answers is not in the document]
Comment (Justkant) [reply apps/ledger-live-mobile/e2e/bridge/server.ts:15]: My bad you totally need that

**Review by Justkant — APPROVED** [review]

**Pull request merged**